### PR TITLE
initial pre-cruise Pioneer MAB deploy csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Initial pre-cruise deploy csvs for testing purposes:
CP11NOSM
CP13SOPM
CP14NEPM
CP14SEPM

- lat/lon for all taken from Table 2-1 in MAB Site Char doc (3210-00007)
- to verify the sensor SNs on the WFPs: Platform_Records/WFPs/Coastal_WFPs and find the correct WFP folder w/ relevant cal csvs, or all the info is captured in one place in the incoming inspection doc (also in Platform_Records) - 3102-00100-00107

- Refer to marine infrastructure doc 1100-00006 (attached), and configs in Pioneer 20 folder in Vault.
[1100-00006_Marine_Infrastructure_Vocabulary_and_Reference_Designators_OOI_2024-01-24_Ver_2-04.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/14300595/1100-00006_Marine_Infrastructure_Vocabulary_and_Reference_Designators_OOI_2024-01-24_Ver_2-04.xlsx)

### CODE REVIEW

**Comment by reedan88** [comment]
I've attached the latitudes/longitudes from 3210-00007
[MAB_latitudes_longitudes.docx](https://github.com/WHOIGit/ooicgsn-asset-management/files/14302196/MAB_latitudes_longitudes.docx)

File: deployment/CP14NEPM_Deploy.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+#AR82,,,,CP14NEPM-RII01-02-ADCPUP021,1,1,2024-02-26T12:00:00,,CGMCP-14NEPM-00001,,,36.0536,-74.7776,,17,300,,,
+AR82,,,,CP14NEPM-RII01-02-ADCPSL010,1,1,2024-02-26T12:00:00,,CGMCP-14NEPM-00001,,CGINS-ADCPSL-18445,36.0536,-74.7776,,281,300,,,
+AR82,,,,CP14NEPM-WFP01-00-WFPENG001,1,1,2024-02-26T12:00:00,,CGMCP-14NEPM-00001,CGVEH-CWFPCP-30001,CGCON-CWFPCP-30001,36.0536,-74.7776,,280,300,,,
+AR82,,,,CP14NEPM-WFP01-01-VEL3DK000,1,1,2024-02-26T12:00:00,,CGMCP-14NEPM-00001,CGVEH-CWFPCP-30001,CGINS-VEL3DK-00468,36.0536,-74.7776,,280,300,,,
+AR82,,,,CP14NEPM-WFP01-02-DOFSTK000,1,1,2024-02-26T12:00:00,,CGMCP-14NEPM-00001,CGVEH-CWFPCP-30001,CGINS-DOFSTK-03506,36.0536,-74.7776,,280,300,,,
+AR82,,,,CP14NEPM-WFP01-03-CTDPFK000,1,1,2024-02-26T12:00:00,,CGMCP-14NEPM-00001,CGVEH-CWFPCP-30001,CGINS-CTDPFK-00184,36.0536,-74.7776,,280,300,,,
+AR82,,,,CP14NEPM-WFP01-04-FLORTK000,1,1,2024-02-26T12:00:00,,CGMCP-14NEPM-00001,CGVEH-CWFPCP-30001,CGINS-FLORTK-01537,36.0536,-74.7776,,280,300,,,
+AR82,,,,CP14NEPM-WFP01-05-PARADK000,1,1,2024-02-26T12:00:00,,CGMCP-14NEPM-00001,CGVEH-CWFPCP-30001,CGINS-PARADK-20608,36.0536,-74.7776,,280,300,,,
```
Comment (reedan88) [inline deployment/CP14NEPM_Deploy.csv:12]: Looks good.

File: deployment/CP14SEPM_Deploy.csv [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+#AR82,,,,CP14SEPM-RII01-02-ADCPUP021,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,,,35.8514,-74.8482,,17,300,,,
+AR82,,,,CP14SEPM-RII01-02-ADCPSL010,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,,CGINS-ADCPSL-21497,35.8514,-74.8482,,281,300,,,
+AR82,,,,CP14SEPM-WFP01-00-WFPENG001,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGCON-CWFPCP-30002,35.8514,-74.8482,,280,300,,,
+AR82,,,,CP14SEPM-WFP01-01-VEL3DK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-VEL3DK-00470,35.8514,-74.8482,,280,300,,,
+AR82,,,,CP14SEPM-WFP01-02-DOFSTK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-DOFSTK-03523,35.8514,-74.8482,,280,300,,,
+AR82,,,,CP14SEPM-WFP01-03-CTDPFK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-CTDPFK-00185,35.8514,-74.8482,,280,300,,,
+AR82,,,,CP14SEPM-WFP01-04-FLORTK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-FLORTK-01538,35.8514,-74.8482,,280,300,,,
+AR82,,,,CP14SEPM-WFP01-05-PARADK000,1,1,2024-02-26T12:00:00,,CGMCP-14SEPM-00001,CGVEH-CWFPCP-30002,CGINS-PARADK-20609,35.8514,-74.8482,,280,300,,,
```
Comment (reedan88) [inline deployment/CP14SEPM_Deploy.csv:12]: Looks good.

**Review by reedan88 — APPROVED** [review]

File: deployment/CP14NEPM_Deploy.csv (hunk 1 above)
Comment (nichole-arm) [inline deployment/CP14NEPM_Deploy.csv:12]: Looks good

File: deployment/CP14SEPM_Deploy.csv (hunk 2 above)
Comment (nichole-arm) [reply deployment/CP14SEPM_Deploy.csv:12]: looks good

**Review by nichole-arm — COMMENTED** [review]
all looks good with the exception of a possible accidental comment on NOSM line 16

File: deployment/CP11NOSM_Deploy.csv [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+AR82,,,,CP11NOSM-MFD36-04-VELPTA000,1,1,2024-02-19T12:00:00,,CGMCP-11NOSM-00001,,CGINS-VELPTA-12711,36.1750,-74.8267,,99,100,,,
+AR82,,,,CP11NOSM-MFD36-05-PCO2WB000,1,1,2024-02-19T12:00:00,,CGMCP-11NOSM-00001,,CGINS-PCO2WB-C0096,36.1750,-74.8267,,99,100,,,
+AR82,,,,CP11NOSM-MFD36-06-PHSEND000,1,1,2024-02-19T12:00:00,,CGMCP-11NOSM-00001,,CGINS-PHSEND-P0069,36.1750,-74.8267,,99,100,,,
+#AR82,,,,CP11NOSM-MFD37-00-DCLENG000,1,1,2024-02-19T12:00:00,,CGMCP-11NOSM-00001,,,36.1750,-74.8267,,99,100,,,
+AR82,,,,CP11NOSM-MFD37-01-OPTAAD000,1,1,2024-02-19T12:00:00,,CGMCP-11NOSM-00001,,CGINS-OPTAAD-00207,36.1750,-74.8267,,99,100,,,
+AR82,,,,CP11NOSM-MFD37-02-ADCPTF000,1,1,2024-02-19T12:00:00,,CGMCP-11NOSM-00001,,CGINS-ADCPTF-19221,36.1750,-74.8267,,99,100,,,
+AR82,,,,CP11NOSM-MFD37-03-CTDBPD000,1,1,2024-02-19T12:00:00,,CGMCP-11NOSM-00001,,CGINS-CTDBPD-50110,36.1750,-74.8267,,99,100,,,
+AR82,,,,CP11NOSM-MFD37-04-DOSTAD000,1,1,2024-02-19T12:00:00,,CGMCP-11NOSM-00001,,CGINS-DOSTAD-00428,36.1750,-74.8267,,99,100,,,
+AR82,,,,CP11NOSM-MFD37-07-ZPLSCC000,1,1,2024-02-19T12:00:00,,CGMCP-11NOSM-00001,,CGINS-ZPLSCC-55101,36.1750,-74.8267,,99,100,,,
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply deployment/CP11NOSM_Deploy.csv:45]: good catch - fixed!

[the remark this answers is not in the document]
Comment (nichole-arm) [reply deployment/CP11NOSM_Deploy.csv:45]: Looks good :)

File: deployment/CP13SOPM_Deploy.csv [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+#AR82,,,,CP13SOPM-RII01-02-ADCPUP021,1,1,2024-02-26T12:00:00,,CGMCP-13SOPM-00001,,,35.7250,-74.8530,,17,100,,,
+AR82,,,,CP13SOPM-RII01-02-ADCPTG010,1,1,2024-02-26T12:00:00,,CGMCP-13SOPM-00001,,CGINS-ADCPTG-24544,35.7250,-74.8530,,82,100,,,
+AR82,,,,CP13SOPM-WFP01-00-WFPENG001,1,1,2024-02-26T12:00:00,,CGMCP-13SOPM-00001,CGVEH-CWFPCP-10305,CGCON-CWFPCP-10305,35.7250,-74.8530,,81,100,,,
+AR82,,,,CP13SOPM-WFP01-01-VEL3DK000,1,1,2024-02-26T12:00:00,,CGMCP-13SOPM-00001,CGVEH-CWFPCP-10305,CGINS-VEL3DK-00034,35.7250,-74.8530,,81,100,,,
+AR82,,,,CP13SOPM-WFP01-02-DOFSTK000,1,1,2024-02-26T12:00:00,,CGMCP-13SOPM-00001,CGVEH-CWFPCP-10305,CGINS-DOFSTK-02628,35.7250,-74.8530,,81,100,,,
+AR82,,,,CP13SOPM-WFP01-03-CTDPFK000,1,1,2024-02-26T12:00:00,,CGMCP-13SOPM-00001,CGVEH-CWFPCP-10305,CGINS-CTDPFK-00128,35.7250,-74.8530,,81,100,,,
+AR82,,,,CP13SOPM-WFP01-04-FLORTK000,1,1,2024-02-26T12:00:00,,CGMCP-13SOPM-00001,CGVEH-CWFPCP-10305,CGINS-FLORTK-01101,35.7250,-74.8530,,81,100,,,
+AR82,,,,CP13SOPM-WFP01-05-PARADK000,1,1,2024-02-26T12:00:00,,CGMCP-13SOPM-00001,CGVEH-CWFPCP-10305,CGINS-PARADK-20466,35.7250,-74.8530,,81,100,,,
```
[the remark this answers is not in the document]
Comment (nichole-arm) [reply deployment/CP13SOPM_Deploy.csv:12]: Update looks good

File: deployment/CP14NEPM_Deploy.csv (hunk 1 above)
Comment (nichole-arm) [reply deployment/CP14NEPM_Deploy.csv:12]: update looks good

File: deployment/CP14SEPM_Deploy.csv (hunk 2 above)
Comment (nichole-arm) [reply deployment/CP14SEPM_Deploy.csv:12]: update looks good

**Review by nichole-arm — COMMENTED** [review]
updates look good :)

File: deployment/CP14NEPM_Deploy.csv (hunk 1 above)
Comment (reedan88) [reply deployment/CP14NEPM_Deploy.csv:12]: Updates look good

File: deployment/CP13SOPM_Deploy.csv (hunk 4 above)
[the remark this answers is not in the document]
Comment (reedan88) [reply deployment/CP13SOPM_Deploy.csv:12]: Update looks good.

File: deployment/CP14SEPM_Deploy.csv (hunk 2 above)
Comment (reedan88) [reply deployment/CP14SEPM_Deploy.csv:12]: Update looks good.

**Review by reedan88 — APPROVED** [review]
Looks good.

**Review by nichole-arm — APPROVED** [review]

**Pull request merged**